Source/WebCore/rendering/RenderImage.cpp

```diff
@@ -741,13 +741,11 @@ ImageDrawResult RenderImage::paintIntoRect(PaintInfo& paintInfo, const FloatRect
     if (!img || img->isNull())
         return ImageDrawResult::DidNothing;
 
-    auto* imageElement = dynamicDowncast<HTMLImageElement>(element());
```

Comment:
This is used below by USE_SYSTEM_PREVIEW) so you have to move that there.

Reply:
moved.